src/rocksdb-types/src/config.rs

```diff
@@ -390,114 +412,27 @@ impl RustType<ProtoRocksDbTuningParameters> for RocksDBTuningParameters {
             stats_log_interval_seconds: proto.stats_log_interval_seconds,
             stats_persist_interval_seconds: proto.stats_persist_interval_seconds,
             point_lookup_block_cache_size_mb: proto.point_lookup_block_cache_size_mb,
+            write_buffer_manager_memory_bytes: proto
+                .write_buffer_manager_memory_bytes
+                .map(usize::cast_from),
+            write_buffer_manager_memory_fraction: proto.write_buffer_manager_memory_fraction,
+            write_buffer_manager_allow_stall: proto.write_buffer_manager_allow_stall,
         })
     }
 }
 
-#[derive(Debug, Clone)]
-pub struct RocksDBDynamicConfig {
-    batch_size: Arc<AtomicUsize>,
-}
-
-impl RocksDBDynamicConfig {
-    pub fn batch_size(&self) -> usize {
-        // SeqCst is probably not required here, but its the easiest to reason about
-        self.batch_size.load(Ordering::SeqCst)
-    }
-}
-
-/// Configurable options for a `RocksDBInstance`. Some can be updated
-/// dynamically, as cloned instances of this object will shared dynamic values.
-#[derive(Debug, Clone)]
-pub struct RocksDBConfig {
```

Comment:
Moved this struct to the `mz-rocksdb` crate, since this would need to use the WriteBufferManager from `rocksdb`.